app/assets/stylesheets/style.scss

```diff
@@ -534,13 +546,14 @@ label:hover {
   .main-menu {
     background: #fbfbfb;
     border-top: 1px solid #e5e5e5;
-    position: absolute;
+    position: fixed;
```

Comment:
Properties should be ordered background, border-top, bottom, height, left, margin-top, position, right, width, z-index